src/tests/ScoreBoard.spec.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import { ScoreBoard } from '../ScoreBoard';
+
+describe('ScoreBoard component', () => {
+  const mockUserMoves = [
+    { name: 'James', moves: 28 },
+    { name: 'Andy', moves: 30 },
+    { name: 'Bill', moves: 32 }
+  ];
+
+  it('Should render correctly', () => {
```

Comment:
use a more descriptive test name. Suggestion- 'should render three items if three scores are saved in localStorage'